packages/storybook8/stories/Examples/CustomButtonInjection/snippets/CustomButtonInjectionTypes.tsx

```diff
@@ -0,0 +1,52 @@
+import { CustomCallControlButtonPlacement, CustomCallControlButtonStrings } from '@azure/communication-react';
+import React from 'react';
+
+export const compositeCanvasContainerStyles = {
+  height: '22rem'
+};
+
+export const addCSS = (css: string): string =>
```

Comment:
can we have this in a more general place since this might be a pattern that will be used at multiple places?